assets/js/modules/analytics/setup.js

```diff
@@ -214,36 +234,36 @@ class AnalyticsSetup extends Component {
 		sendAnalyticsTrackingEvent( 'analytics_setup', 'profile_change', selectValue );
 	}
 
-	async getAccounts() {
-		const { isEditing } = this.props;
-		const {
-			errorCode,
-			useSnippet,
-		} = this.state;
+	async getAccounts( existingTagData = false ) {
 		let {
 			selectedAccount,
 			selectedProperty,
 			selectedProfile,
 		} = this.state;
+		const {
+			isEditing,
+			onSettingsPage,
+		} = this.props;
+		const {
+			errorCode,
+			useSnippet,
+		} = this.state;
 		let newState = {};
 
 		try {
-			const responseData = await data.get( 'modules', 'analytics', 'get-accounts', {}, false );
+			// Send existing tag data to get account.
+			const queryArgs = existingTagData ? {
+				account: existingTagData.account,
+				property: existingTagData.property,
```

Comment:
Please rename these to something more clear to what they are, e.g. `existingAccountId` & `existingPropertyId`.